addons/website_sale/views/templates.xml

```diff
@@ -2979,51 +3001,35 @@
-                        <span>Order</span>
-                        <span t-field="order.name"/>
-                        <t t-if="order.state == 'sale'">
-                            <i class="fa fa-check-circle ms-1"/>
-                        </t>
-                    </em>
+                    <span>Order</span>
+                    <span t-field="order.name"/>
                 </h5>
             </div>
+            <div class="oe_structure mt-3" id="oe_structure_website_sale_confirmation_2"/>
             <t t-if="request.env['res.users']._get_signup_invitation_scope() == 'b2c' and request.website.is_public_user()">
                 <p class="alert alert-info mt-3" role="status">
                     <a role="button" t-att-href="order.partner_id.signup_prepare() and order.partner_id.with_context(relative_url=True)._get_signup_url()" class="btn btn-primary">Sign Up</a>
-                    to follow your order.
+                    <span class="align-middle">to follow your order.</span>
                 </p>
             </t>
             <div class="oe_structure clearfix mt-3" id="oe_structure_website_sale_confirmation_1"/>
-            <h4 class="text-start mt-3">Payment Information</h4>
-            <table class="table">
-                <tbody>
-                    <tr>
-                        <td colspan="2" class="ps-0">
-                            <t t-esc="tx_sudo.provider_id.name" />
-                        </td>
-                        <td class="text-end pe-0" width="100">
-                            <strong>Total:</strong>
-                        </td>
-                        <td class="text-end pe-0" width="100">
-                            <strong t-field="tx_sudo.amount" t-options="{'widget': 'monetary', 'display_currency': order.currency_id}" />
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
             <t t-call="website_sale.payment_confirmation_status"/>
-            <div class="card mt-3">
-                <div class="card-body">
-                    <t t-set="same_shipping" t-value="bool(order.partner_shipping_id == order.partner_invoice_id or only_services)" />
-                    <div><b>Billing <t t-if="same_shipping and not only_services"> &amp; Delivery</t>: </b><span t-esc='order.partner_invoice_id' t-options="dict(widget='contact', fields=['address'], no_marker=True, separator=', ')" class="address-inline"/></div>
-                    <div t-if="not same_shipping and not only_services" groups="account.group_delivery_invoice_address"><b>Delivery: </b><span t-esc='order.partner_shipping_id' t-options="dict(widget='contact', fields=['address'], no_marker=True, separator=', ')"  class="address-inline"/></div>
-                </div>
+            <div class="mt-3">
+                <t t-call="website_sale.address_on_checkout"/>
             </div>
-            <div class="oe_structure mt-3" id="oe_structure_website_sale_confirmation_2"/>
```

Comment:
Why do you remove that one?
this kind of hooks are very useful for custom views/modules, removing them will very probably break stuff so unless really needed/requested, we should keep those :thinking:

Reply:
I didn't remove that just move it [here](https://github.com/odoo/odoo/pull/198731/files#diff-635040c730ccbb8e4f6f2bf6472cf48a0cc9b8a3d488289dabdfce8096811915R3021) for this point 
![image](https://github.com/user-attachments/assets/c2343e9a-38ac-4565-8cfd-5dae22a0dd62)

Reply:
Well, a bit strange that the second hookable part is now before the first one.
What should we do about that @chevalierv ?